tests/libtest/test_ibm_hci_nodes.py

```
@@ -0,0 +1,300 @@
+
+        Ensures all nodes are powered on and ready after each test
+        Uses the IBMHCINode.restart_nodes_by_stop_and_start_teardown() method
+        """
+
+        def finalizer():
+            logger.info("Teardown: Ensuring all nodes are powered on and ready")
+            try:
+                ibm_hci = IBMHCINode()
+                # Use the teardown method from IBMHCINode class
+                ibm_hci.restart_nodes_by_stop_and_start_teardown()
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)